pkg/render/compliance.go

```diff
@@ -136,6 +136,13 @@ func (c *complianceComponent) Objects() []runtime.Object {
 			c.complianceServerService(),
 			c.complianceServerDeployment(),
 		)
+	} else {
+		// We still need a serviceaccount in the managed cluster for RBAC requests.
+		complianceObjs = append(complianceObjs,
+			c.complianceServerServiceAccount(),
```

Comment:
Since the complianceServerServiceAccount() and complianceServerClusterRoleBinding() need to be in every cluster, could you just add them to the original list of objs please?

Reply:
Alina argued that this might be more clear/maintainable as it is explicitly defined here, so I went along with that. I'm fine either way

Reply:
The intention of this block was to put in objects that were different in Manged / Management cases. I don't think it really improves readability, personally I think it promotes confusion as to why we put these in two places, especially if we need to remove / change the service account or role binding, somebody might think that they should only change it for one of the cases and leave it the same for the other.

Reply:
I suggested this so we always have the service account, role and binding together to avoid confusion on how they are applied.

Reply:
What exactly do you think somebody will be confused about though?

Reply:
Well I agree that it might be weird to see the sa, cr and crb without a deployment. Someone not knowing what is going on might think that this is a mistake. I now clarified this with a comment, so I think we are safe to close this discussion.